src/Logging/MonologErrorHandler.php

```diff
@@ -92,7 +93,13 @@ public function start()
         }
 
         foreach ($loggers as $logger) {
-            MonologHandler::register($logger);
+            // Log deprecation warnings as WARNING, not NOTICE
+            // see https://github.com/Seldaek/monolog/blob/1.x/doc/01-usage.md#log-levels
+            $errorLevelMap = [
+                E_DEPRECATED => LogLevel::WARNING,
+                E_USER_DEPRECATED => LogLevel::WARNING,
+            ];
+            MonologHandler::register($logger, $errorLevelMap);
```

Comment:
See the main issue for discussion about this - tl;dr monolog themselves say deprecations should be logged as warnings, but then they log them as notice by default. That seems silly, so we're logging them as warnings.